src/gui/MainWindow.ui

```diff
@@ -545,6 +546,11 @@
     <string>Show TOTP</string>
    </property>
   </action>
+  <action name="actionEntryTotpDisplayKey">
+   <property name="text">
+    <string>Show TOTP key</string>
```

Comment:
Something like "Export TOTP settings" would be better here, since someone could confuse the key as the generated code.